client/src/components/SearchPage/Header/Header.js

```diff
@@ -55,14 +55,12 @@ const Header = (props) => {
         <React.Fragment>
             <Navbar className={classes.navbar}>
                 <Nav className={classes.navlinks}>
-                    <Nav.Link href="#">
-                        <img src={Menu} className={classes.apps} />
-                    </Nav.Link>
                     <Nav.Link
                         href={`https://github.com/login/oauth/authorize?client_id=${CLIENT_ID}&scope=user`}
                     >
                         <div className={classes.login}>
-                            <img className={classes.github} src={GithubIcon} />
+                            {/* <img className={classes.github} src={GithubIcon} /> */}
+                            <LogoGithub onClick={() => alert('Hi!')} fontSize="35px" color="white   " />
```

Comment:
Is an onClick function required here? If it's not, then I guess you can remove it too